code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -340,16 +340,32 @@
 	if(hasHUD(user,"security"))
 		var/perpname = get_visible_name(TRUE)
 		var/criminal = "None"
+		var/commentLatest = "None"
 
 		if(perpname)
 			for(var/datum/data/record/E in data_core.general)
 				if(E.fields["name"] == perpname)
 					for(var/datum/data/record/R in data_core.security)
 						if(R.fields["id"] == E.fields["id"])
 							criminal = R.fields["criminal"]
+			if(commentLatest)
+				for(var/datum/data/record/E in data_core.general)
+					if(E.fields["name"] == perpname)
+						for(var/datum/data/record/R in data_core.security)
+							if(R.fields["id"] == E.fields["id"])
+								if(LAZYLEN(R.fields["comments"]))
+									for(var/c in R.fields["comments"])
+										commentLatest = LAZYACCESS(R.fields["comments"],(R.fields["comments"].len)) //Get the latest entry from the comment log
+								else
+									commentLatest = "No entries."
+
 			var/criminal_status = hasHUD(user, "read_only_security") ? "\[[criminal]\]" : "<a href='?src=[UID()];criminal=1'>\[[criminal]\]</a>"
+			var/commentLatest_check = "[commentLatest]"

```

Comment:
This is a copy of the value. If you want [comment] as text you should use \[comment\]

You can also put this directly in the msg like the other info things do

Reply:
True that! The difference between machine learning and monkey learning, the monkey learns just slower...

Thanks for these!